app/main/views/agreements.py

```diff
@@ -237,3 +237,56 @@ def put_signed_framework_agreement_on_hold(agreement_id):
         return jsonify(message="Database Error: {0}".format(e)), 400
 
     return jsonify(agreement=framework_agreement.serialize())
+
+
+@main.route('/agreements/<int:agreement_id>/countersign', methods=['POST'])
+def countersign_agreement(agreement_id):
```

Comment:
Yeah, I think this is probably a good distinction to make, there doesn't seem to be another way of referring to that particular set of agreements.

Where is the frontend checking for a file path?